release-notes/20221103-ovn-sbctl.md

```diff
@@ -0,0 +1,4 @@
+<!-- markdownlint-disable MD041 -->
+The `subctl gather` command now includes the output from `ovn-sbctl show`
+which has the mapping of chassis-id to hostname mapping that can be used
```

Comment:
Maybe you should delete one of the 'mapping' ? 
Something like : 

"which has the chassis-id to hostname mapping that can be used .... "